airflow/www/static/js/dag/details/index.tsx

```diff
@@ -301,7 +302,7 @@ const Details = ({
           <TabPanel height="100%">
             {isDag && <DagContent />}
             {isDagRun && <DagRunContent runId={runId} />}
-            {isTaskInstance && (
+            {shouldShowDetailsTab && (
```

Comment:
```suggestion
            {!!runId && !!taskId (
```
Let's just do it in a single line since we don't use the variable anywhere else.

Reply:
resolved :)